### PR TITLE
Gatsby source

### LINKED ISSUE #148: Decrease code repetition (request for enhancement)
Currently we have reached a stage where we have enough engagement and new contributors are adding content/code.

The lists maintained are now beginning to become difficult to maintain (for example, managing the width of the tile). See 1dae173470190edf8263ecef8350e50d2928baa3 for example.

It may be the right time to consider bringing in a framework/generator.

The site is [served by Mozilla's Participation infrastructure](https://discourse.mozilla.org/t/migration-of-mozilla-india-blog-and-website-to-participation-infrastructure/19152) through github pages and therefore whatever works on github pages would be the preferred way to get this done. (Think jekyll, etc)

I would very much love if someone other than me can take this up (plus guarantee support from my side if we are switching to jekyll)

### PR DESCRIPTION
**Pull request by harshil1712**
Fixes #148

### CODE REVIEW

File: data/regionalgroups.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        "instagram":null,
+        "meetup":null,
+        "facebook":null
+    },
+    {
+        "name":"Mozilla Gujarat",
+        "website":"http://mozillagujarat.org",
+        "telegram":"https://t.me/joinchat/CpXrs0J9FOfBVAB4TTEzAw",
+        "twitter":"https://twitter.com/mozguj",
+        "GitHub":"https://github.com/mozguj",
```
Comment (asdofindia) [inline data/regionalgroups.json]: Can we have github also in all small case

**Comment by asdofindia** [comment]
I think as soon as we revert the license, readme, etc and re-enable the hamburger button in mobile view, we can merge this and then fix the smaller bugs later.

**Comment by harshil1712** [comment]
Hey, sure. I was wondering we should shift this PR to some other branch, so that others can take up the remaining work time, like changing License, updating the README.md file, adding the CONTRIBUTORS.md file and other things. This will help other contributors to contribute as well, and speed up the process. What do you think?

**Comment by obitodarky** [comment]
I agree with @harshil1712. Moving this to another branch will also provide other contributors to test the site locally and contribute accordingly. We can then merge to master branch for deployment.

**Comment by asdofindia** [comment]
Ah, a branch on Mozilla India itself, you mean. Okay. That sounds reasonable. (I was thinking people could collaborate on your repository, but this maybe easier). Let me take care of it.

**Comment by asdofindia** [comment]
Okay, pushed to https://github.com/MozillaIndia/mozillaindia.github.io/tree/gatsby

**Pull request closed without merging**